<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-08-10T18:36:16Z" name="Default suite" finished-at="2024-08-10T18:36:47Z" duration-ms="31642">
    <groups>
    </groups>
    <test started-at="2024-08-10T18:36:16Z" name="Default test" finished-at="2024-08-10T18:36:47Z" duration-ms="31642">
      <class name="com.qa.tests.Verifyloginssecondtestcase">
        <test-method is-config="true" signature="initreports()[pri:0, instance:com.qa.tests.Verifyloginssecondtestcase@457c9034]" started-at="2024-08-11T00:06:15Z" name="initreports" finished-at="2024-08-11T00:06:16Z" duration-ms="243" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initreports -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.qa.tests.Verifyloginssecondtestcase@457c9034]" started-at="2024-08-11T00:06:16Z" name="setUp" finished-at="2024-08-11T00:06:22Z" duration-ms="6843" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="Validationoftabsafterlogin()[pri:0, instance:com.qa.tests.Verifyloginssecondtestcase@457c9034]" started-at="2024-08-11T00:06:22Z" name="Validationoftabsafterlogin" description="Verify Tabs functionality" finished-at="2024-08-11T00:06:47Z" duration-ms="24455" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Welcome, SATISH KUMAR!Not matched with Support This Project availble from the UI expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Welcome, SATISH KUMAR!Not matched with Support This Project availble from the UI expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at com.qa.utilities.Commonfunctions.VerifyUI(Commonfunctions.java:53)
	at com.qa.tests.Verifyloginssecondtestcase.Validationoftabsafterlogin(Verifyloginssecondtestcase.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Validationoftabsafterlogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.tests.Verifyloginssecondtestcase@457c9034]" started-at="2024-08-11T00:06:47Z" name="tearDown" finished-at="2024-08-11T00:06:47Z" duration-ms="323" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="fleshreports()[pri:0, instance:com.qa.tests.Verifyloginssecondtestcase@457c9034]" started-at="2024-08-11T00:06:47Z" name="fleshreports" finished-at="2024-08-11T00:06:47Z" duration-ms="254" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fleshreports -->
      </class> <!-- com.qa.tests.Verifyloginssecondtestcase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
